notebooks/data_viz_easy/raw/ex1.ipynb

```diff
@@ -232,16 +109,16 @@
    "source": [
     "## Step 2: Review the data\n",
     "\n",
-    "Print the last 5 rows of the data."
```

Comment:
i was tentatively planning to teach `head` & `tail`, but planned to revisit once writing the tutorial (if it's too long, i'll switch back to just `head`). 

my primary concern here (and why i decided to go with `tail` [at least tentatively] was that ... at the top of the notebook, i show a screen capture / picture of the first five rows.  i think this image is useful, because it shows the students what the CSV file looks like in Excel (so, when they load the dataset with pandas, it's easier to understand what has happened, and to trust / understand the work that they do in Pandas).

drawback (re: providing that image) is that if i ask questions about the first five rows, they don't have to use `head` at all and can just work from the image.  so, i asked questions about the `tail` 🤔

Reply:
Seems fine. I also think it's reasonably to tell them in the question `Use a python command to print the first 5 rows ...`

If they want to look at the image on the top, that's up to them. But at least they'll know what they are being asked to do.